opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -705,19 +705,22 @@ class _Span(Span):
 
 class Tracer(trace_api.Tracer):
```

Comment:
Don't we need a change in `get_tracer` in `TracerProvider` as well?

Reply:
Referring to [this](https://github.com/open-telemetry/opentelemetry-python/pull/1295/files#diff-b51d60b85f22187922aa5324512b7ef69fc259a932ffb33ac9bc011cb50debcdL891) or something more?

Reply:
Oops, must've missed this.